jcl/src/java.management/share/classes/java/lang/management/ThreadInfo.java

```diff
@@ -121,119 +91,46 @@ private ThreadInfo(long threadIdVal, String threadNameVal,
 			long lockOwnerIdVal, String lockOwnerNameVal,
 			StackTraceElement[] stackTraceVal, LockInfo lockInfo,
 			MonitorInfo[] lockedMonitors, LockInfo[] lockedSynchronizers
-			/*[IF Sidecar19-SE]*/
+/*[IF Sidecar19-SE]*/
 			, boolean daemon, int priority
-			/*[ENDIF]*/
+/*[ENDIF]*/
 	) {
-		this.threadId = threadIdVal;
-		this.threadName = threadNameVal;
-		this.threadState = threadStateVal;
-		this.suspended = suspendedVal;
-		this.inNative = inNativeVal;
-		this.blockedCount = blockedCountVal;
-		this.blockedTime = blockedTimeVal;
-		this.waitedCount = waitedCountVal;
-		this.waitedTime = waitedTimeVal;
-		this.lockName = lockNameVal;
-		this.lockOwnerId = lockOwnerIdVal;
-		this.lockOwnerName = lockOwnerNameVal;
-		this.stackTraces = stackTraceVal;
-		this.lockInfo = lockInfo;
-		this.lockedMonitors = lockedMonitors;
-		this.lockedSynchronizers = lockedSynchronizers;
-		/*[IF Sidecar19-SE]*/
-		this.daemon = daemon;
-		this.priority = priority;
-		/*[ENDIF]*/
-	}
-
-	/**
-	 * @param thread
-	 * @param state
-	 * @param isSuspended
-	 * @param isInNative
-	 * @param blockedCount
-	 * @param blockedTime
-	 * @param waitedCount
-	 * @param waitedTime
-	 * @param stackTrace
-	 * @param blockingMonitor
-	 * @param lockOwner
-	 */
-	private ThreadInfo(Thread thread, long nativeId, int state, boolean isSuspended,
```

Comment:
Is this constructor still used?

Reply:
No.  It was used by the native code, which not creates the base object instead.

Reply:
Please remove it then.